imcsdk/imccoreutils.py

```diff
@@ -285,10 +285,17 @@ def extract_molist_from_method_response(method_response,
     """
 
     mo_list = []
-    if len(method_response.out_configs.child) == 0:
-        return mo_list
+    if hasattr(method_response, "out_config"):
+        if len(method_response.out_config.child) == 0:
+            return mo_list
+    else:
+        if len(method_response.out_configs.child) == 0:
```

Comment:
@prvashis I stand corrected.. I think configResolveClass in ucsm also returns outConfigs, as it could return multiple Mos, even if one class was mentioned.